src/gluonts/dataset/artificial/generate_synthetic.py

```diff
@@ -1,241 +0,0 @@
-        2: "WED",
-        3: "THU",
-        4: "FRI",
-        5: "SAT",
-        6: "SUN",
-    }
-    for i in range(len(time_series)):
-        data = time_series[i]
-        timestamp = pd.Timestamp(data[FieldName.START])
-        freq_week_start = freq
-        if freq_week_start == "W":
-            freq_week_start = f"W-{week_dict[timestamp.weekday()]}"
-        timestamp = pd.Timestamp(data[FieldName.START], freq=freq_week_start)
-        item_id = int(data[FieldName.ITEM_ID])
-        for j, target in enumerate(data[FieldName.TARGET]):
-            # Using convention that there are no missing values before the
-            # start date
-            if is_missing and j != 0 and j % num_missing == 0:
-                timestamp += 1
-                continue  # Skip every 4th entry
-            else:
-                timestamp_row = timestamp
-                if freq in ["W", "D", "M"]:
-                    timestamp_row = timestamp.date()
-                row = [item_id, timestamp_row, target]
-                # Check if related time series is present
-                if FieldName.FEAT_DYNAMIC_REAL in data.keys():
-                    for feat_dynamic_real in data[FieldName.FEAT_DYNAMIC_REAL]:
-                        row.append(feat_dynamic_real[j])
-                csv_writer.writerow(row)
-                timestamp += 1
-
-
-def generate_sf2(
-    filename: str, time_series: List, is_missing: bool, num_missing: int
-) -> None:
-    #  This function generates the test and train json files which will be
-    #  converted to csv format
-    if not os.path.exists(os.path.dirname(filename)):
-        os.makedirs(os.path.dirname(filename))
-    with open(filename, "w") as json_file:
-        for ts in time_series:
-            if is_missing:
-                target = []  # type: List
-                # For Forecast don't output feat_static_cat and
-                # feat_static_real
-                for j, val in enumerate(ts[FieldName.TARGET]):
-                    # only add ones that are not missing
-                    if j != 0 and j % num_missing == 0:
-                        target.append(None)
-                    else:
-                        target.append(val)
-                ts[FieldName.TARGET] = target
-            ts.pop(FieldName.FEAT_STATIC_CAT, None)
-            ts.pop(FieldName.FEAT_STATIC_REAL, None)
-            # Chop features in training set
-            if (
-                FieldName.FEAT_DYNAMIC_REAL in ts.keys()
-                and "train" in filename
-            ):
-                # TODO: Fix for missing values
-                for i, feat_dynamic_real in enumerate(
-                    ts[FieldName.FEAT_DYNAMIC_REAL]
-                ):
-                    ts[FieldName.FEAT_DYNAMIC_REAL][i] = feat_dynamic_real[
-                        : len(ts[FieldName.TARGET])
-                    ]
-            json.dump(ts, json_file)
-            json_file.write("\n")
-
-
-def generate_sf2s_and_csv(
```

Comment:
@lostella @jgasthaus Is this something we still want to use? I don't think we have much usage for csv files.